.github/workflows/check-merge-conflict.yml

```diff
@@ -0,0 +1,13 @@
+name: Check For Merge Conflict
+
+on:
+  workflow_call:
```

Comment:
Can this (in the consumer projects) also run when the PR is not touched? A PR can get conflicted even if one doesn't touch it, but somebody else pushing to the target branch.

I guess it can: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=For%20readability%2C%20also%20comment%20the%20version%20there.

Reply:
Yeah, this runs on all PRs when one is touched (as experienced)

Reply:
How is that possible? In [the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) I don't see `pull_request` triggering then (the trigger used in the OSOCE workflow). Also:

> Conversely, workflows with the pull_request_target event will run even if the pull request has a merge conflict.

Reply:
The action runs on the PR touched, but will affect every open PR in the repo. When I opened the PR for this branch in the main OSOCE repo, for example your pull request for OSOCE-56 wasn't touched in 14 days, and got flagged for a merge conflict.

Reply:
I don't understand how `actions-label-merge-conflict` really works, then. Does it actually go through all the PRs in a repo,  not just the current one? Because then I can understand it doing this, given we run the workflow in OSOCE on `dev` pushes too, which most of the PRs are targeting. If indeed, then please document what kind of triggers we recommend, because then `pull_request` is not suitable alone (as it is in Workflows.md) and there needs to be a `push` trigger configured too.

(I just noticed that in my first comment I didn't link what I wanted: https://github.com/eps1lon/actions-label-merge-conflict#example-usage)

Reply:
My best guess is that yes. It's not documented anywhere, but empirical evidence is pointing to that. I'll add the recommended triggers to the doc then.

Reply:
You can also check out the action's code, not to have to guess ;).